CMakeLists.txt

```diff
@@ -593,17 +596,24 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
     # statically link slang into the CIRCTImportVerilog library, but seems to be
     # harder than it ought to be.
     set_property(
-      GLOBAL APPEND PROPERTY CIRCT_EXPORTS slang_slang unordered_dense fmt)
+      GLOBAL APPEND PROPERTY CIRCT_EXPORTS slang_slang fmt)
     
     # Disable the installation of headers coming from third-party libraries. We
     # won't use those APIs directly. Just make them static libraries for the sake
     # of running slang normally.
     set_target_properties(fmt PROPERTIES PUBLIC_HEADER "")
-    set_target_properties(unordered_dense PROPERTIES PUBLIC_HEADER "")
 
-    install(TARGETS slang_slang unordered_dense fmt EXPORT CIRCTTargets)
+    install(TARGETS slang_slang fmt EXPORT CIRCTTargets)
+
+    # Match the behavior of slang_slang, which installs its own vendored
+    # boost_unordered if it does not a system-wide boost installation.
+    find_package(Boost 1.82.0 QUIET)
```

Comment:
If boost is escaping into the API, we should probably be picking up the boost from the slang build, or at least guaranteeing it's the same version without hardcoding it here.